apps/condo/domains/acquiring/utils/taskSchema/index.js

```diff
@@ -458,6 +460,80 @@ async function sendTomorrowPaymentNotificationSafely (context, recurrentPaymentC
     }
 }
 
+async function sendTomorrowPaymentNoReceiptsNotificationSafely (context, recurrentPaymentContext) {
+    if (isNil(recurrentPaymentContext) || isNil(get(recurrentPaymentContext, 'id'))) throw new Error('invalid recurrentPaymentContext argument')
```

Comment:
maybe add logger too

Reply:
we have logger at the top level of the payment processing, all throwed errors will appears there